runtime/parsing_functions.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "runtime/kphp_core.h"
+#include <regex>
+#include <iostream>
+#include <fstream>
+#include <string>
```

Comment:
In most cases it's unsafe to use std utilities in runtime until you 100% sure you're doing.

The problem is kphp runtime can always be interrupted by signal (script timeout for example), which leads to `swapcontext` back to network context and script termination WITHOUT calling any destructors. It will cause memory leaks (because std often uses heap memory) and a lot of other problems.
In your case using any of these dependencies is wrong and unsafe.
See other comments for more detailed explanation for each usage.